devops/openshift/forms-deploy.yml

```diff
@@ -164,11 +164,11 @@ objects:
     spec:
       scaleTargetRef:
         apiVersion: apps/v1
-        kind: ReplicationController
-        name: ${NAME}-rc
+        kind: DeploymentConfig
+        name: ${NAME}
       minReplicas: "${{REPLICAS}}"
-      maxReplicas: 10
-      targetCPUUtilizationPercentage: 80
+      maxReplicas: 6
+      targetCPUUtilizationPercentage: 50
```

Comment:
I had this doubt yesterday when you are demoing, if the targetCPUUtilizationPercentage is cannot be raised above 50%. Can we change the CPU Request to a higher value like 1, and see if something changes??

Reply:
As I understood it was tested with 80% but there were many failed requests because it took long for the replicas to be up and running, so with 50% it had better results.

Reply:
@guru-aot the CPU can go beyond 50% and it will because the new replicas will take a couple of minutes to start getting the extra load. As @andrepestana-aot mentioned, when leaving at 80% it was degrading too much the success responses.